nomenclature/definitions/variable/technology/technologies.yaml

```diff
@@ -3,263 +3,78 @@
-                power plants
-   unit: GW
-
-Capacity|Electricity|Ocean:
-   description: Total installed electricity generation capacity of ocean power
-                plants
-   unit: GW
-
-Capacity|Electricity|Oil:
-   description: Total installed electricity generation capacity of oil power
-                plants
-   unit: GW
-
-Capacity|Electricity|Other:
-   description: Total installed electricity generation capacity of other power
-                plants
-   unit: GW
-
-Capacity|Electricity|Solar:
-   description: Total installed electricity generation capacity of solar power
-                installations
-   unit: GW
-
-Capacity|Electricity|Solar|CSP:
-   description: Total installed electricity generation capacity of CSP plants
-   unit: GW
-
-Capacity|Electricity|Solar|PV:
-   description: Total installed electricity generation capacity of PV
-                installations
-   unit: GW
-
-Capacity|Electricity|Wind:
-   description: Total installed electricity generation capacity of wind power
-                installations
+   
+Capacity|Electricity|<Fuel>:
+   description: Total installed electricity generation capacity from <this fuel>
    unit: GW
 
-Capacity|Electricity|Wind|Offshore:
-   description: Total installed electricity generation capacity of offshore
-                wind power installations
+Capacity|Residential and Commercial Heat:
+   description: Total installed heat generation capacity for the residential and commercial sectors
    unit: GW
 
-Capacity|Electricity|Wind|Onshore:
-   description: Total installed electricity generation capacity of onshore wind
-                power installations
+Capacity|Residential and Commercial Heat|<Fuel>:
+   description: Total installed heat generation capacity for the residential and commercial sectors from <this fuel>
    unit: GW
    
-Capacity|Electricity|Hydrogen:
-   description: Total installed electricity generation capacity from hydrogen
+Capacity|Industrial Heat:
+   description: Total installed heat generation capacity for the industrial sector
    unit: GW
-
-Capacity|Electricity|Hydrogen|OCGT:
-   description: Total installed electricity generation capacity from hydrogen of Open Cycle Gas Turbines
-                plants
+   
+Capacity|Industrial Heat|<Fuel>:
+   description: Total installed heat generation capacity for the industrial sector from <this fuel>
    unit: GW
 
-Capacity|Electricity|Hydrogen|Fuel Cell:
-   description: Total installed electricity generation capacity from hydrogen of fuelcell plants
-   unit: GW  
-
-Capital Cost|Electricity|Biomass|w/ CCS:
-   description: Capital cost of a new biomass power plant with CCS
+Capital Cost|Electricity|<Fuel>:
+   description: Capital cost for <this fuel>
    unit: US$2010/kW
 
-Capital Cost|Electricity|Biomass|w/o CCS:
-   description: Capital cost of a new biomass power plant w/o CCS
+Capital Cost|Residential and Commercial Heat|<Fuel>:
```

Comment:
I'm wondering if restructuring this as `Capital Cost|Heat|Residential and Commercial|*` would be a smarter approach?

Reply:
Might not be harmful and is maybe more easily readable. I changed the heating technologies accordingly.